packages/protocol/test-sol/stability/SortedOracles.mento.t.sol

```diff
@@ -0,0 +1,659 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Was this test used a template for `SortedOracles.t.sol` or is it still needed?

If it's still neeted, consider renaming the test contract to something along the lines of `SortedOraclesMentoTest`

Reply:
They are actually different, just for now, I'd leave both to make sure nothing is borken.